shell/browser/api/electron_api_safe_storage.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2021 Slack Technologies, Inc.
+// Use of this source code is governed by the MIT license that can be
+// found in the LICENSE file.
+
+#ifndef SHELL_BROWSER_API_ELECTRON_API_SAFE_STORAGE_H_
+#define SHELL_BROWSER_API_ELECTRON_API_SAFE_STORAGE_H_
+
+#include <string>
+
+#include "gin/handle.h"
+#include "gin/wrappable.h"
+#include "shell/browser/event_emitter_mixin.h"
+#include "shell/common/gin_helper/cleaned_up_at_exit.h"
+
+namespace electron {
+
+namespace api {
+
+class SafeStorage : public gin::Wrappable<SafeStorage>,
```

Comment:
There's not really any reason to use a `gin::Wrappable` here as the methods are all static methods. In fact, you don't need the `SafeStorage` class, or even this header file at all!